quickstarts/microsoft.web/web-app-asp-app-on-asev3-create/modules/site.bicep

```diff
@@ -1,58 +1,35 @@
-@description('Required. App service prefix.')
-param appNamePrefix string = 'app-2'
+@description('App service prefix.')
+param appName string = 'GEN-UNIQUE'
```

Comment:
GEN values are only valid in the parameters file, not in the template - just remove the defaults since this is a module

Reply:
done